<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_ip_plane</class>
 <widget class="QDialog" name="Dialog_ip_plane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>9</y>
     <width>1491</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>航班号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="et_p_number"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>起飞地</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="et_d_city"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>到达地</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="et_l_city"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>价格</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="et_price"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>折扣</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="et_discount"/>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>可载人数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="et_people"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>起飞时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="t_d">
        <property name="displayFormat">
         <string>yyyy/M/d H:mm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>到达时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="t_l">
        <property name="displayFormat">
         <string>yyyy/M/d H:mm</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_add">
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_decide">
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
